### PR TITLE
Total not skill issue PR

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Rearranges teleporter to be little bit less of a mess and moves all drums to the ammo disk (including now carbine and regular rifle.)
Removes ROE from vouchers and vendor, but now you can print it off the disk.
Makes Blackguard have brace penalty to not be better /longarm/ in every way
Replaces Sterlki in armory (still can print it) with Watchtower since it seems more appropriate with the color scheme, which in it of itself loses 2 recoil and 0.1 mod on it's damage.
Rearranged ammo disk with now drums (mentioned above) and weird things like carbine rubbers being short mags while rest long removed.
Rearranges the ammo boxes to be next to eachother in the armory
guhhhhh

<hr>
</details>

## Changelog
:cl:
del: ROE from BS vouchers
tweak: Teleporter list (Order of items in it)
tweak: Blackshield ammo disk (drums and magazines only)
tweak: Blackshield armory Strelki for Watchtower and rearranged ammo boxes on shelves
balance: Blackguard being better Longarm
/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Please fully detail your changelogs.

File: code/game/machinery/vendor/weapon_kit_vendors.dm [hunk 1]
```diff
@@ -134,13 +129,12 @@
...
 	var/items = list(
-					"Stallion Kit" = /obj/item/storage/box/bs_kit/stallion,
 					"Colt Kit" = /obj/item/storage/box/bs_kit/colt,
-					"Makarov Kit" = /obj/item/storage/box/bs_kit/makarov,
+					"Makarov Kit" = /obj/item/storage/box/bs_kit/makarov,		
+					"Stallion Kit" = /obj/item/storage/box/bs_kit/stallion,
 					"Cowboy Kit" = /obj/item/storage/box/bs_kit/rex10,
 					"Pilgrim Kit" = /obj/item/storage/box/bs_kit/pilgrim,
-					"Sawn-Off Shotgun Kit" = /obj/item/storage/box/bs_kit/sawn_shotgun,
-					"Martin Kit" = /obj/item/storage/box/bs_kit/martin)
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/weapon_kit_vendors.dm:143]: Martin should stay

File: code/modules/projectiles/guns/projectile/automatic/blackguard.dm [hunk 2]
```diff
@@ -12,14 +12,15 @@
 	origin_tech = list(TECH_COMBAT = 7, TECH_MATERIAL = 2)
 	slot_flags = SLOT_BACK
 	load_method = SINGLE_CASING|MAGAZINE
-	mag_well = MAG_WELL_HRIFLE|MAG_WELL_DRUM
+	mag_well = MAG_WELL_HRIFLE //Haha, no
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/blackguard.dm:15]: Add back the drum and read the desk, this gun was made to take the drum

**Comment by Trilbyspaceclone** [comment]
No drums in the teleporter

File: code/modules/projectiles/guns/projectile/automatic/blackguard.dm (hunk 2 above)
Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/automatic/blackguard.dm:15]: Agree with Trilby. Unnecessary change for a weapon that is one of the few able to take these drums and that was designed with this in mind

**Comment by Churro55** [comment]
The Armory change is bad, in general. No reason for it to be changed like that; it makes less effective use of space and it removes not only weapons but ammunition too.
What's the reasoning behind removing loadout options too? "Martin doesn't get used" isn't a good reason, but you removed the DB as well.

File: code/modules/projectiles/guns/projectile/automatic/blackguard.dm (hunk 2 above)
Comment (Lamasmaster) [reply code/modules/projectiles/guns/projectile/automatic/blackguard.dm]: Reverted but not the brace penalty, if long arm has one this should too

File: code/game/machinery/vendor/weapon_kit_vendors.dm (hunk 1 above)
Comment (Lamasmaster) [reply code/game/machinery/vendor/weapon_kit_vendors.dm]: I guess if you want it to stay just feels weird to have 2 of them

**Comment by Lamasmaster** [comment]
Redone some of the changes, nobody removed DB learn to read the code it just got shuffled under the comment.

**Comment by Churro55** [comment]
Oh okay, let me update the list then.
You removed:
-1 Blackguard
-The Strelki and all it's ammunition
-ALL  of SLAP
-The 257 Box
-All non-lethal grenades
-The Martin Kit (Not anymore)
-The DB Kit (Not anymore)
-The Roe Kit (Not anymore)

**Comment by Lamasmaster** [comment]
Let me update it for you since you clearly are trying to just gaslight me at this point or are blind whichever might be the case
See anything removed ? I don't, thanks.
guhhhhh

**Comment by Churro55** [comment]
Okay, I'll update the list.
-Nothing was removed.
-Strelki swapped with Watchtower.

Thank you.

**Comment by lolman360** [comment]
roe kit still being removed :/

**Comment by ThePainkiller** [comment]
Drums were added to the teleporter at a fair price not long ago, why are they being removed from the teleporter again?

File: code/game/machinery/excelsior/blackshield_teleporter.dm [hunk 3]
```diff
@@ -34,47 +34,44 @@ var/global/blackshield_max_energy //Maximaum combined energy of all teleporters
 	var/list/parts_list = list(
 		//Ammo
 		/obj/item/ammo_casing/flare/old = 10, //Rng colour for cheap! - dont want to add a spam list of every type ever
+		/obj/item/ammo_magazine/highcap_pistol_35 = 15, //Idea with this is your sending up the mag more then ammo
 		/obj/item/ammo_magazine/smg_35 = 35,
```
Comment (Trilbyspaceclone) [inline code/game/machinery/excelsior/blackshield_teleporter.dm:38]:
```suggestion
		/obj/item/ammo_magazine/smg_35 = 35,
		/obj/item/ammo_magazine/highcap_pistol_35/drum = 125, //Needed for PPSH
```

**Comment by SouthyOsmanli** [comment]
> Drums were added to the teleporter at a fair price not long ago, why are they being removed from the teleporter again?

Only the 7.5 should be. I added the .408 - but that's only because on the Exotic disk the .408 drum takes 0 license points to print empty. Only reason I've been against 7.5 on there is it takes 1 license to print empty, 2 for full. So it'd be a no-brainer just to non-stop use the teleporter for cost/efficiency.

**Comment by ThePainkiller** [comment]
> I asked for only the 7.5 one to just be removed from tele and put on disk, so there's a 'rarity' to it to encourage going out and getting them from others; but you can still make them yourself for license-points. Which I think is fair - Marshals have the same.

I'm talking about .408 drums being deleted from the teleporter

**Comment by Lamasmaster** [comment]
Well due to how we were testing ROE yesterday, I wouldn't call it as much as useless anymore but more why isn't this a disk weapon to be honest since it still doesn't fit the color pattern of blackshield. The .408 drums definetly weren't fairly priced not that this makes it any better since they only cost like 6 metal in lathe now and trilby said all drums (except .35 I guess because I hate regular PPSH not taking sticks like PPV) so all drums.

**Pull request merged**